modules/installation-uninstall-vsphere.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/uninstalling-cluster-vsphere.adoc
+
+[id="installation-uninstall-vsphere_{context}"]
+= Removing a cluster from vSphere
+
+You can remove a cluster that you installed on vSphere.
+
+.Procedure
+
+* From the vSphere Client, delete each virtual machine that your cluster uses.
```

Comment:
After deleting the VMs, the user can also delete the folder created to store the VMs.